fimsauthz-api/utm-roles.yml

```diff
@@ -0,0 +1,131 @@
+# UTM roles and, when applicatble, their scopes.
+#   This is the master data for the Role and Scopes for the
+#   FIMS Authorization Server.
+#   Its schema is defined as a domain in swgger spec.
+#
+
+UTM_AUTHORITY:
+    description: |
+      A small set of trusted users who manage identities of all core UTM
+      components such as USSs.
+    identity_owner: itself
```

Comment:
is "identity_owner" the right term?  is this really indicating who "owns" the "identity"?  Seems like this is the entity that vets another entity for this role, correct?  identity_validator?  identity_manager?  Open to other suggestions.

Reply:
Agreed, should change this .  Perhaps identity_authority? Or identity_assigned_by.

Priya and I asked DickH and he used  "Authority. :